namespace.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+export LUAXDP_GW=enp0s1 # replace it with your gateway interface
+echo 1 > /proc/sys/net/ipv4/ip_forward
+ip netns add luaxdp
+ip netns exec luaxdp ip link set lo up
+ip link add luaxdp0 type veth peer name luaxdp1
+ip link set luaxdp1 netns luaxdp
+ip addr add 10.0.0.1/24 dev luaxdp0
+ip netns exec luaxdp ip addr add 10.0.0.2/24 dev luaxdp1
+ip link set luaxdp0 up
+ip netns exec luaxdp ip link set luaxdp1 up
+iptables -A FORWARD -o $LUAXDP_GW -i luaxdp0 -j ACCEPT
+iptables -A FORWARD -i $LUAXDP_GW -o luaxdp0 -j ACCEPT
+iptables -t nat -A POSTROUTING -s 10.0.0.2/24 -o $LUAXDP_GW -j MASQUERADE
+ip netns exec luaxdp ip route add default via 10.0.0.1
+mkdir -p /etc/netns/luaxdp
+echo "nameserver 1.1.1.1" > /etc/netns/luaxdp/resolv.conf
+echo 'alias ns="sudo ip netns exec luaxdp su - $USER"' > .nsrc
+echo "# usage:\n$ source .nsrc\n$ ns\n$ curl https://wheel.to"
```

Comment:
nao precisa um `echo -e` aqui pra imprimir o `\n`?

Reply:
tb pensava isso.. mas ao rodar, imprimia o "-e".. to achando que eh mopa do `sh` padrao que link para `dash` no ubuntu vs `bash`

see https://stackoverflow.com/questions/57126531/why-will-echo-e-test-print-e-test

posso trocar para `#!/bin/bash` e usar `-e` ou deixar como esta.. o que acha?

Reply:
entao, eu acho que o problema eh que o `echo` nao eh portavel ne. cada lugar roda de um jeito. em geral, se voce usar um linter de sh script geralmente ele sugere substituir echo por printf.

Reply:
troquei por printf